shared/js/ui/base/safari-ui-wrapper.es6.js

```diff
@@ -94,13 +94,21 @@ let openExtensionPage = (path) => {
         path = path.substr(1)
     }
 
-    let tab = safari.application.activeBrowserWindow.openTab()
-    tab.url = getExtensionURL(path)
+    let url = getExtensionURL(path)
+
+    if (context === 'popup') {
+        let tab = safari.application.activeBrowserWindow.openTab()
+        tab.url = url
+        safari.self.hide()
+    } else {
+        // note: this will only work if this is happening as a direct response
+        // to a user click - otherwise it'll be blocked by Safari's popup blocker
+        window.open(url, '_blank')
+    }
```

Comment:
A bit more of an explanation about what's happening here:

In Safari, the popup runs in a different context from the extension pages (e.g. options, feedback) so the two have access to completely different APIs. Which is why `safari.application.activeBrowserWindow` doesn't currently work from the options page.